src/utils.js

```diff
@@ -0,0 +1,38 @@
+export const cleanMovies = (movies) => {
+    return movies.map(movie => {
+        return {
+            id: movie.id,
+            title: movie.title,
+            poster_path: movie.poster_path
+        }
+    })
+}
+
+export const cleanMovie = (selectedMovie) => {
+    let {backdrop_path, title, average_rating,
+        release_date, overview, genres, budget,
+         revenue, runtime, tagline} = selectedMovie;
+    average_rating = average_rating.toFixed(2);
+    let reorder = release_date.split('-')
+    let year = reorder.shift();
+    reorder.splice(2, 0, year)
+    release_date = reorder.join('-');
```

Comment:
Surely there is a better way to do this array reordering

Reply:
There might be, but I think yours is fine. I would consider making a function like `restructureDate` and invoking it from this function. Would help to pull some of this logic out of here.